docs/content/UnderlineNav2.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: UnderlineNav
```

Comment:
Since we already have an "UnderlineNav" that this is replacing, I think this title needs to be something like "UnderlineNav v2" to differentiate it.